akka-bbb-fsesl/build.sbt

```diff
@@ -28,8 +28,8 @@ val compileSettings = Seq(
 )
 
 resolvers ++= Seq(
-  "spray repo" at "http://repo.spray.io/",
-  "blindside-repos" at "http://blindside.googlecode.com/svn/repository/"
+  ("spray repo" at "http://repo.spray.io/").withAllowInsecureProtocol(true),
+  ("blindside-repos" at "http://blindside.googlecode.com/svn/repository/").withAllowInsecureProtocol(true)
```

Comment:
this repo seems to be an old / not existing repo? Also, please do not use repositories without TLS and trusted certificate in the build chain.